RestaurantReservation.API/Profiles/CustomerProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using AutoMapper;
+using RestaurantReservation.API.Models.Customers;
+using RestaurantReservation.Db.Models.Entities;
+
+namespace RestaurantReservation.API.Profiles
+{
+    public class CustomerProfile : Profile
```

Comment:
have you looked into other libraries than automapper ?

Reply:
yes ihave worked with AutoMapper
but I’ve also explored other libraries like Mapster this alternative are faster and lighter, and i sometimes prefer them for simpler mapping or when performance is critical
however i ended up using AutoMapper because I was more familiar with it